webapi/src/main/scala/org/knora/webapi/store/triplestore/upgrade/plugins/UpgradePluginPR2255.scala

```diff
@@ -61,6 +63,18 @@ class UpgradePluginPR2255(log: Logger) extends UpgradePlugin {
 
 object ProjectsIrisToChange {
   val prefix = "http://rdfh.ch/projects/"
+
+  // KEEP it for reference, not a part of plugin and used only for replacement of
+  // IRIs existing in DSP-API repo
+  val onlyFoundInRepo: Map[String, String] = Map(
+    "0113" -> "bFURQB6zR1ugDFQL7EZy9Q",
+    "0666" -> "cXxbOh4xRce3uLUyXHujfg",
+    // beol in v1 test data
+    "0801" -> "bL0y8GRuTUiFmvF1oXbeFQ",
+    // dokubib in v1 test data
+    "00FE" -> "oIjhUsZmQLuJ0VMGvJ2pfg"
+  )
```

Comment:
What do you mean?

Reply:
something like that:

```scala
/*
 * "0113" -> "bFURQB6zR1ugDFQL7EZy9Q",
 * "0666" -> "cXxbOh4xRce3uLUyXHujfg",
 * // beol in v1 test data
 * "0801" -> "bL0y8GRuTUiFmvF1oXbeFQ",
 * // dokubib in v1 test data
 * "00FE" -> "oIjhUsZmQLuJ0VMGvJ2pfg"
*/
```

so that we definitely never get a `XY is never used` compiler warning... but it really doesn't matter

Reply:
Forgot to answer and change it to as you suggested. At the end I'm not sure if we should move it to some documentation in the repo on Confluence, but living here is most probably can't be omitted if someone touches this code. I have have change it in another PR. Also what was weird IDE didn't complain to it being unused...